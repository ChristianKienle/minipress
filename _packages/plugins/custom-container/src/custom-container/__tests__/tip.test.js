// @ts-check
const tip = require('./../tip')
const createMd = require('./../../../create-md')

describe('hi', () => {
  it('do', () => {
    const md = createMd()
    md.use(tip)
    const html = md.render('\n\n::: tip bla\ntest\n:::\n\n')
    expect(html).toMatchSnapshot()

  })
})
