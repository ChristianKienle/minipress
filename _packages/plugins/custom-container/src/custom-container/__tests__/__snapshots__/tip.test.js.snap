// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hi do 1`] = `
"<MiniTip title=\\"bla\\"><p>hihi</p>
</MiniTip>
"
`;
